rskj-core/src/test/java/org/ethereum/datasource/KeyValueDataSourceTest.java

```diff
@@ -115,7 +115,7 @@ public void putNullValue() {
         keyValueDataSource.put(randomKey, null);
     }
 
-    @Test(expected = IllegalArgumentException.class)
+    @Test(expected = RuntimeException.class)
```

Comment:
hmm.. does this mean that LevelBD will continue throwing `IllegalArgumentException`, but RocksDB - `RuntimeException` ? I think that these two db implementations should be compatible and behave the same way (probably to throw `IllegalArgumentException` as it was before). What is your thought on this?

Reply:
Done